src/wp-includes/blocks.php

```diff
@@ -151,23 +151,27 @@ function register_block_style_handle( $metadata, $field_name ) {
 	$style_handle = generate_block_asset_handle( $metadata['name'], $field_name );
 	$block_dir    = dirname( $metadata['file'] );
 	$style_file   = realpath( "$block_dir/$style_path" );
-	$version      = file_exists( $style_file ) ? filemtime( $style_file ) : false;
-	$result       = wp_register_style(
-		$style_handle,
-		$style_uri,
-		array(),
-		$version
-	);
-	if ( file_exists( str_replace( '.css', '-rtl.css', $style_file ) ) ) {
-		wp_style_add_data( $style_handle, 'rtl', 'replace' );
-	}
+
 	if ( file_exists( $style_file ) ) {
+		$result = wp_register_style(
+			$style_handle,
+			$style_uri,
+			array(),
+			filemtime( $style_file )
```

Comment:
Ideally, we should not use `filetime()` for production sites. Is there a reason why we can't use the WordPress version here? When `version` is omitted here, the default should be the current WordPress Core version.

Reply:
Can block metadata pass a `version`? I can't find the metadata documented anywhere within the inline documentation, so it's hard to know for sure what will be included.

As a spin-off, I'd love to get at least the required fields, and maybe the officially recognized optional ones documented.

Reply:
It was the case before, so we should keep the existing handling.

Reply:
Block metadata is documented at https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#block-api

Should we link it in a particular place in the code?

There is no `version` in the definition. For JavaScript files, we read additional data like the `version` (unique hash based on the file content) from the `.asset.php` file generated by webpack. We don't have anything like that for CSS files, but we might consider extending the build functionality to align with JS.